context.go

```diff
@@ -224,7 +224,7 @@ func (c *Context) Fill() {
 	validators := c.Config.GetValidators(txx...)
 	c.NextConsensus = c.Config.GetConsensusAddress(validators...)
 
-	if now := uint32(c.Config.Timer.Now().Unix()); now > c.Timestamp {
+	if now := uint64(c.Config.Timer.Now().Unix()); now > c.Timestamp {
```

Comment:
`....UnixNano()/1000`, or maybe just `UnixNano` and let the user of it decide.

Reply:
Or even `/1000000` because we have millisecond precision being used for 3.0, but again, let's keep nanoseconds in `Context`.

Reply:
Added `/1000000`

Reply:
I think we can implement a `getTimestamp(*timer.Timer) uint64` or `toTimestamp(time.Time) uint64`

Reply:
Added `(c *Context) getTimestamp() uint64` as we don't use this method anywhere except from `Fill()`. So, is it appropriate for our Context?

Reply:
It is used in `send.go` and `check.go` as well. But these are timestamps for payloads, I see.